_content/articles/buddhism-and-speciesism_sciberras-colette.md

```diff
@@ -0,0 +1,20 @@
+---
+title: "Buddhism and Speciesism: on the Misapplication of Western Concepts to Buddhist Beliefs"
+authors:
+  - "Colette Sciberras"
+external_url: "https://blogs.dickinson.edu/buddhistethics/files/2010/05/sciberras-article.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1uLxVxzBup_4CuCkL1AiSMqZ2Aaedsoxa/view?usp=sharing"
+course: animals
+tags:
+  - ethics
+  - sutta
```

Comment:
No need for `sutta` (hopefully most articles about early Buddhism reference the suttas!)  But this definitely needs either `west` or `dialogue`